analyzer/runtime/extract.go

```diff
@@ -453,7 +453,7 @@ func ExtractRound(blockHeader nodeapi.RuntimeBlockHeader, txrs []nodeapi.Runtime
 			// Early versions of runtimes didn't emit a GasUsed event.
 			if (txr.Result.IsUnknown() || txr.Result.IsSuccess()) && tx != nil {
 				// Treat as if it used all the gas.
-				logger.Info("tx didn't emit a core.GasUsed event, assuming it used max allowed gas", "tx_hash", txr.Tx.Hash(), "assumed_gas_used", tx.AuthInfo.Fee.Gas)
+				logger.Debug("tx didn't emit a core.GasUsed event, assuming it used max allowed gas", "tx_hash", txr.Tx.Hash(), "assumed_gas_used", tx.AuthInfo.Fee.Gas)
```

Comment:
:hear_no_evil:

Reply:
It's expected prior to a certain height (was thinking of hardcoding it, but it depends on the runtime and chain), and nearly impossible after the height/version that introduced core.GasUsed events. So it's not providing much value. Should be `Debug` in the first case and `Error` or `Warn` in the latter, really.

Reply:
maybe someday we can have a field saying what upgrade each block is from